src/thematic/vegetatie/svp-401_vegetatieopname-terrestrisch/svp-401_vegetatieopname-terrestrisch.Rmd

```diff
@@ -0,0 +1,189 @@
+---
+title: Klassieke vegetatieopname in een proefvlak aan de hand van visuele inschattingen van bedekking van soorten in (semi-)terrestrische vegetatie
+subtitle: 
+author: Els De Bie
+date: 2016-07-19
+output: 
+  bookdown::html_document2:
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+<!--
+Instructies bij opmaak van veldprotocol:
+
+Hieronder vind je de verschillende onderdelen van een veldprotocol en een beschrijving wat er bij elk onderdeel vermeld moet worden. 
+Niet alle onderdelen zijn altijd van toepassing. Laat in die gevallen de titelhoofdingen staan en vermeld expliciet “niet van toepassing”.
+Probeer het veldprotocol generiek te maken in de mate van mogelijk en probeer te vermijden dat het veldprotocol enkel specifiek voor een bepaald project bruikbaar is. 
+Je kan het protocol generiek houden door bepaalde variabelen, zoals bijvoorbeeld de grootte van een proefvlak of de diepte van een bodembemonstering, niet vast te leggen in het protocol. 
+In bijlage kan je dan een tabel toevoegen waarin je per project aangeeft welke waardes gekozen moeten worden voor die variabelen. 
+Indien een bepaald onderdeel niet van toepassing is, laat je de hoofding staan en vermeld je hieronder ‘Niet van toepassing’. Op die manier is de nummering van alle protocols dezelfde.
+-->
+
+```{r setup, include=FALSE}
+library(knitr)
+opts_chunk$set(
+  echo = FALSE, 
+  eval = TRUE,
+  dpi = 300,
+  fig.width = 150 / 25.4,
+  fig.height = 100 / 25.4,
+  out.width = '100%',
+  warning = FALSE,
+  error = TRUE,
+  message = FALSE
+)
+```
+
+
+```{r metadata}
+# pas deze tabel aan
+metadata <- data.frame(
```

Comment:
add this sections to the yaml

```
params:
    reviewers:
         - 
           naam: "Hans Van Calster"
           orcid: "xxxx-xxxx-xxxx-xxxx"
         - 
           naam: "Lieve Vriens"
           orcid: "xxxx-xxxx-xxxx-xxxx"
   documentbeheerder: "Hans Van Calster"
```

the information will available in the document as the `params` object (a named list).